frontend/toolbar/white_toolbar_mobile.css

```diff
@@ -0,0 +1,30 @@
+.white-toolbar-mobile {
+  background-color: white;
+  box-shadow: 0px 3px 6px grey;
+  height: 60px;
+  width: 100%;
+  font-size: 20px;
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: 20% 50% auto;
+  align-items: center;
+}
+
+.toolbar-icon-alignment {
+    font-size: 2.5em;
+    line-height: inherit !important;
+}
+
+.edit-inst-toolbar-title {
```

Comment:
use the property text-transform to set the title to make sure it will always be uppercase